### PR TITLE
restore jetty-home

### PR DESCRIPTION
**Pull request by olamy**
- get jetty-home back
- fix modules declaration

### CODE REVIEW

**Review by gregw — COMMENTED** [review]
We don't need to do PRs on jetty-12 just yet... at least not for simple things like this.

HOWEVER, please don't re-enable jetty-home until it can actually build something (and not break the build).

**Comment by olamy** [comment]
> We don't need to do PRs on jetty-12 just yet... at least not for simple things like this.
> 
> HOWEVER, please don't re-enable jetty-home until it can actually build something (and not break the build).

@gregw it's green and assemble the jars the PR is to be sure it's green
see original branch (except flaky test_ https://jenkins.webtide.net/job/jetty.project/job/jetty-12.0.x-jetty-home/

File: jetty-home/pom.xml [hunk 1]
```diff
@@ -236,6 +236,56 @@
...
+                  <classifier>sources</classifier>
+                </artifactItem>
+                <artifactItem>
+                  <groupId>org.eclipse.jetty.toolchain</groupId>
+                  <artifactId>jetty-jakarta-servlet-api</artifactId>
+                  <!-- FIXME MUST BE A PROPERTY -->
+                  <version>5.0.2</version>
+                  <classifier>sources</classifier>
+                </artifactItem>
+              </artifactItems>
```
Comment (gregw) [inline jetty-home/pom.xml:285]: Oh cool, you can bring in two different versions.

**Review by gregw — APPROVED** [review]
Let's give this a go

**Pull request merged**